eng/Version.Details.xml

```diff
@@ -19,13 +19,13 @@
     </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="6.0.0-beta.21167.3">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="6.0.0-beta.21175.1">
```

Comment:
Why is this in your PR now?

Reply:
I think I mistakenly rebased off of darc-main-0fade34f-2916-414b-9f7c-5b5d8ffdef60.  How do I un-rebase off that commit?